.github/workflows/main.yml

```diff
@@ -18,39 +18,45 @@ on:
 
 jobs:
   test:
-    uses: viamrobotics/rdk/.github/workflows/test.yml@main
+    uses: viamrobotics/rdk/.github/workflows/test.yml@static-build
     secrets:
       MONGODB_TEST_OUTPUT_URI: ${{ secrets.MONGODB_TEST_OUTPUT_URI }}
 
   slam_integration_test:
-    uses: viamrobotics/rdk/.github/workflows/slam-integration-test.yml@main
+    uses: viamrobotics/rdk/.github/workflows/slam-integration-test.yml@static-build
 
   appimage:
     needs: test
-    uses: viamrobotics/rdk/.github/workflows/appimage.yml@main
+    uses: viamrobotics/rdk/.github/workflows/appimage.yml@static-build
     secrets:
       GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
 
-  npm_publish:
-    uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@main
+  staticbuild:
     needs: test
+    uses: viamrobotics/rdk/.github/workflows/staticbuild.yml@static-build
     secrets:
-      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-  slack-workflow-status:
-    if: ${{ failure() }}
-    name: Post Workflow Status To Slack
-    needs:
-      - test
-      - appimage
-    runs-on: ubuntu-latest
-    permissions:
-      actions: 'read'
-    steps:
-      - name: Slack Workflow Notification
-        uses: Gamesight/slack-workflow-status@master
-        with:
-          repo_token: ${{secrets.GITHUB_TOKEN}}
-          slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
-          channel: '#team-devops'
-          name: 'Workflow Status'
+      GCP_CREDENTIALS: ${{ secrets.GCP_CREDENTIALS }}
+
+  # npm_publish:
+  #   uses: viamrobotics/rdk/.github/workflows/npm-publish.yml@static-build
+  #   needs: test
+  #   secrets:
+  #     NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  # slack-workflow-status:
+  #   if: ${{ failure() }}
+  #   name: Post Workflow Status To Slack
+  #   needs:
+  #     - test
+  #     - appimage
+  #   runs-on: ubuntu-latest
+  #   permissions:
+  #     actions: 'read'
+  #   steps:
+  #     - name: Slack Workflow Notification
+  #       uses: Gamesight/slack-workflow-status@master
+  #       with:
+  #         repo_token: ${{secrets.GITHUB_TOKEN}}
+  #         slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
+  #         channel: '#team-devops'
+  #         name: 'Workflow Status'
```

Comment:
To all be uncommented before going live. Just didn't want this triggering during testing.